application.js

```diff
@@ -8,46 +8,66 @@
 function runChallenges() {
 
   // Ex 1. Read the content of the email input
-  var email = null;  // TODO: replace null, keep the email variable.
+  var email = document.getElementById('email').value;  // TODO: replace null, keep the email variable.
 
 
   // Ex 2. Fill the content of the email input with your email
+  document.getElementById('email').value = "k.perachon@gmail.com";
 
 
 
   // Ex 3. Replace the email hint (next to the input) with 'This is my email now'
   //       The text should be emphasized using a <strong> tag
-
+  document.getElementById('email-hint').innerHTML = "<strong>This is my email now</strong>";
```

Comment:
Same as above

    var emailHint = $('#email-hint');
    var text = $('<strong>').text("This is my email now");
    text.appendTo(emailHint);